components/script/dom/webidls/CompositionEvent.webidl

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+/*
+ * The origin of this IDL file is
+ * https://w3c.github.io/uievents/#idl-compositionevent
+ *
+ */
+
+// https://w3c.github.io/uievents/#idl-compositionevent
+[Constructor(DOMString type, optional CompositionEventInit eventInitDict)]
+interface CompositionEvent : UIEvent {
```

Comment:
Please add a `Pref="dom.compositionevent.enabled"` annotation and add a new preference to resources/prefs.json.